web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -163,6 +173,9 @@ export default defineComponent({
       incrementalSaveRecord(root.$route.params.id);
       if (valid === false) {
         errorClick(0);
+        samplesValid.value = false;
+      } else {
+        samplesValid.value = true;
```

Comment:
@pkalita-lbl I noticed that `samplesValid` wasn't getting updated anywhere. Does this make sense as the place to update that value? Are there any other places in the `HarmonizerView` that would need to update the value as well?

Reply:
I think that's the right place to update `samplesValid` but I think keying off of the `valid` value might not be quite right. The value of `valid` in that function is referring to the _current_ tab. I think it would be safer to check the `invalidCells` object, which holds the validation state for _all_ the tabs.

Reply:
`harmonizerApi.ts::validate` only validates the current tab. There is currently no way to validate all tabs at once (we can chat about the technical reasons why, but it's probably more detail than what's needed here). 

But tab A is validated when it is the current tab and the user selects tab B. So the validation state of all the non-current tabs _should_ be accurate. The big caveat is the case where you have tabs A, B, and C and the user has never clicked on tab C, for example. I wonder if a bit of state to track which tabs have been viewed is warranted, and that the Submit button becomes enabled if there are no validation errors _and_ all tabs have been viewed.

Reply:
cd9516b replaces `samplesValid` with a new piece of state called `tabsValidated`, which tracks validation across all tabs. If data is edited in the data harmonizer, it is marked as invalid. Now, in order to enable the submit button, a user has to click validate on all tabs.

Reply:
c33b381 adds new UI features to indicate what the user needs to do in order to submit.

https://user-images.githubusercontent.com/7085625/228962535-cd5c4af1-708d-47e7-a2d0-fc410de1c3f3.mp4

Reply:
Looks good to me!